### PR TITLE
:pencil: Update a few typos

### PR DESCRIPTION
**Pull request by jefftriplett**
I spotted a minor typo and then updated it to keep the dates consistent in the opening.

### CODE REVIEW

File: _posts/2024-01-17-announcing-djangocon-us-2023.md [hunk 1]
```diff
@@ -8,14 +8,13 @@ title: Code, test, ship it! DjangoCon US 2024 has been announced!
 
 Hello everyone, we have our dates! September 22nd-27th.
 
-The conference will have tutorials on Sunday, September 22, talks from Monday,
-September 23rd, through Wednesday, September 25th, and sprints on Thursday and Friday September 26th and 27th, 2024.
+The conference will have tutorials on Sunday, September 22nd, talks from Monday, September 23rd through Wednesday, September 25th, and sprints on Thursday and Friday, September 26th and 27th, 2024.
```
Comment (jefftriplett) [inline _posts/2024-01-17-announcing-djangocon-us-2023.md:11]: @drewbrew is there a consistent date-style here you care about?

Comment (drewbrew) [reply _posts/2024-01-17-announcing-djangocon-us-2023.md:11]: Your change looks good to me

**Review by tim-schilling — APPROVED** [review]
I'm not sure about the "This year," comma. The rest of them make sense to me.

**Comment by jefftriplett** [comment]
> I'm not sure about the "This year," comma. The rest of them make sense to me.

Grammarly and google docs both flagged it for me. It's not the end of the world to remove the comma if you want it back. The missing "the" is what stood out to me.

**Comment by tim-schilling** [comment]
That's fair enough. Grammarly likes its commas.

**Comment by jefftriplett** [comment]
Merging this since it's a typo fix.

**Comment by drewbrew** [comment]
> I'm not sure about the "This year," comma. The rest of them make sense to me.

It's one of those where you're technically _fine_ to skip it but it's better to have it.

**Pull request merged**